docs/includes/pairlists.md

```diff
@@ -164,6 +165,29 @@ If the trading range over the last 10 days is <1%, remove the pair from the whit
 !!! Tip
     This Filter can be used to automatically remove stable coin pairs, which have a very low trading range, and are therefore extremely difficult to trade with profit.
 
+#### VolatilityFilter
```

Comment:
Maybe include the wikipedia-link somewhere in this section?

Also (i've never used volatility as such), is there a range the values can be in? 
like from 0.0-1.0 (is this a ratio / percentage value)? 
I'm pretty sure that would simplify finding a "desired" configuration...